src/Workspaces/Core/Portable/Serialization/SerializerService.cs

```diff
@@ -6,16 +6,19 @@
 using System.Collections.Concurrent;
 using System.Composition;
 using System.Linq;
+using System.Runtime.Versioning;
 using System.Threading;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.Internal.Log;
-using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.Serialization;
 
+#if NETCOREAPP
+[SupportedOSPlatform("windows")]
+#endif
```

Comment:
matches code we have on TemporaryStorageService, and prevents warnings from showing up. Effectively, the serializer service is windows only (since that's the only place we have certainty that mmfs work). But that's ok. We only need the mmfs for serialization for host/oop syncing. And that's currently VS-only (which means windows only). So it's fine for this code to be solely for Windows.